utrecht.tf

```diff
@@ -101,4 +101,9 @@ resource "github_repository_collaborators" "utrecht" {
     permission = "maintain"
     team_id    = github_team.frameless-maintainer.slug
   }
+
+  team {
+    permission = "push"
+    team_id    = github_team.logius-maintainer.slug
```

Comment:
@remypar5 ik denk dat logius-committer genoeg is

Reply:
We doen bewust `logius-maintainer` omdat `logius-committer` ook leden van de Logius community bevat. De afnemers van Logius die ons willen helpen. Die willen we niet in anderen hun repo laten wroeten. `logius-maintainer` bevat alleen leden van team LUX.

Reply:
Mee eens, deze maintainers hebben dan geen maintainer rechten.